### DISCUSSION TITLE
Ability to view trackpoint location and time stamp

### DISCUSSION THREAD

**Discussion by leep70** [osmandapp/OsmAnd#11630] [Maps Related]
I currently use Garmin MapSource to view GPX files on my PC.  MapSource allows me to scroll through a list of location points (trackpoints) while interactively seeing those location points on a map which is "zoomable". This allows me to see exactly where I was at a given time. A screenshot example of that functionality appears here:  [https://www.leepirtle.com/gps/gpsquestion01.jpg].

MapSource is no longer supported nor is it usable in iOS thus I am searching for a "mobile" alternative. A friend suggested that I post the question to OsmAnd.

Might someone direct me to software that offers that functionality or a possible "workaround"?
Thanks,
Lee

**Comment by karl-01** [comment]
Try out "RouteConverterForWindows".
If you load any compatible track, it will show you each track-point on a map and in a list the corresponding position.

Karl

**Reply by leep70** [reply]
Hello Karl,
Thank you for that suggestion. It looks like a perfect substitute for Garmin MapSource and if/when MapSource no longer functions on my computer I will definitely use "RouteConverterForWindows". I very much appreciate you telling me about it. However, my current need is to have software that will run on an iOS iPad Pro. Again, I appreciate your information.
